python/dgl/data/fakenews.py

```diff
@@ -20,18 +20,10 @@ class FakeNewsDataset(DGLBuiltinDataset):
     who retweeted the root news. Besides, the node features are encoded
     user historical tweets using different pretrained language models:
 
-    - bert: the 768-dimensional node feature composed of Twitter user
-    historical tweets encoded by the bert-as-service
-
-    - content: the 310-dimensional node feature composed of a
-    300-dimensional “spacy” vector plus a 10-dimensional
-    “profile” vector
-
-    - profile: the 10-dimensional node feature composed of ten Twitter
-    user profile attributes.
-
-    - spacy: the 300-dimensional node feature composed of Twitter user
-    historical tweets encoded by the spaCy word2vec encoder.
+    - bert: the 768-dimensional node feature composed of Twitter user historical tweets encoded by the bert-as-service
+    - content: the 310-dimensional node feature composed of a 300-dimensional “spacy” vector plus a 10-dimensional “profile” vector
+    - profile: the 10-dimensional node feature composed of ten Twitter user profile attributes.
+    - spacy: the 300-dimensional node feature composed of Twitter user historical tweets encoded by the spaCy word2vec encoder.
```

Comment:
Have you tried locally rendering the document?

Reply:
Actually not, but I followed the style of existing docstring. If you can provide the instruction on how to render the document locally, I will be happy to do so.

Reply:
You can follow the instructions in `docs/README.md`.  Note that this builds MXNet, PyTorch and Tensorflow documentations in one shot.

Reply:
Thanks a lot!